docs/UserGuide.adoc

```diff
@@ -1065,7 +1066,37 @@ Format: `setrate RATE`
 Examples:
 
 * `setrate 25` +
-Sets the current hourly rate to 25$.
+Sets the current hourly rate to $25.00.
+
+=== Statistics Window
+
+==== Generate Statistic Report : `report`
+
+You can use this command to generate a report to display information such as:
+
+* A breakdown and summary of completed sessions
+* The number of hours of each type of completed sessions
+* A breakdown of your student's ratings
+
+Optionally, you can specify a module code. If a module code is specified, the report generated will only include data from the specified module.
+
```

Comment:
Could you help me add a line below this to say that pressing ESC closes the window? thanks

Reply:
Alright